modules/applying-the-latest-internal-certificates.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="applying-the-latest-internal-certificates_{context}"]
+= Applying the latest internal certificates
+
+By manually restarting the pods of the affected cluster, you can apply the latest certificates and prevent downtime.
```

Comment:
I think we should make it clearer that this is an exceptional case (so we do not contradict our own statement that the feature requires no user intervention). Something like:
> If a Secured Cluster shows a soon-to-expire credential warning, even though auto-refresh is enabled,...
 
If you think it helps you could use the [NOTE] or [IMPORTANT] formatting.